elevation/doc/doc.md

```diff
@@ -1,17 +1,16 @@
 ```js script
-import '@divriots/dockit-core/sass-showcases';
-import { html } from '~/md-layout';
+import '@divriots/dockit-core/css-showcases';
 ```
 
 # Elevation
 
 ## box-shadow
 
-```html:html
-<dockit-sass-showcases
+```html story
+<dockit-css-showcases
   css-props-prefix="--shadow"
   component-class="box"
   style-key="box-shadow"
   checkered-background="false"
-></dockit-sass-showcases>
+></dockit-css-showcases>
```

Comment:
elevation is not visible in the dark mode

one might think it's because of `mdjs-layout` typography which has too dark background which is not compatible with the DS, but I think the shadows themselves are not dark compatible, so I'd recommend to consider adding 2 sets of shadows for light and dark separately

if you feel like it's because of `mdjs-layout` or how I set it up, we should discuss it
I have a well-thought vision here and prefer to have it discussed thoroughly before changing to smth different

![image](https://user-images.githubusercontent.com/137844/151864780-4c668927-e981-4544-8756-7b421ea5c9b0.png)

Reply:
Yep, I agree on having two different set of shadows for light/dark modes. Thanks 👍